projects/example/src/main.ts

```diff
@@ -3,6 +3,7 @@ import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 
 import { AppModule } from './app/app.module';
 import { environment } from './environments/environment';
+import '@angular/compiler';
```

Comment:
why?

Reply:
Workaround because our app is build with old version of `@angular-devkit/build-angular`